etc/kayobe/kolla.yml

```diff
@@ -560,6 +560,8 @@ kolla_build_args: {}
 kolla_overcloud_inventory_pass_through_host_vars_extra:
   - stackhpc_gpu_data
   - gpu_group_map
+  - "stackhpc_radosgw_usage_exporter_frontend_port"
+  - "stackhpc_radosgw_usage_exporter_backend_port"
```

Comment:
nit: Can we either drop the quotes or quote everything in the list? It's going to look odd if there's no pattern